### PR TITLE
2 new levels have been added

### PR DESCRIPTION
**Pull request by shoc71**
- these levels have been skinned with layers and differently colored platforms (platform overlayed by a drawing)

### CODE REVIEW

**Comment by slysnivy** [comment]
@shoc71 The conflicts are an easy fix, but the real problem is that there are 3 levels called EasyLevel22. Would it be ok with you if I renamed these?

File: dont stop now/assets/levels.txt [hunk 1]
```diff
@@ -1328,10 +1328,218 @@ self.win_zones =  [
...
+        pygame.draw.rect(screen, (139, 0, 0), [794, 222, 13, 43])
+]
+
+self.win_zones =  [
+        pygame.draw.rect(screen, (47, 237, 237), [0, 462, 13, 68])
+]
+
 ===
 
+EasyLevel22(57, 45, 1)
```
Comment (slysnivy) [inline dont stop now/assets/levels.txt:1380]: EasyLevel22 occurrence 1

File: dont stop now/assets/levels.txt [hunk 2]
```diff
@@ -1328,10 +1328,218 @@ self.win_zones =  [
...
+        pygame.draw.rect(screen, (139, 0, 0), [12, 500, 1057, 12])
+]
+
+self.win_zones =  [
+        pygame.draw.rect(screen, (47, 237, 237), [102, 495, 66, 17])
+]
+
 ===
 
+EasyLevel22(24, 67, 1)
```
Comment (slysnivy) [inline dont stop now/assets/levels.txt:1461]: EasyLevel occurrence 2

File: dont stop now/assets/levels.txt (hunk 1 above)
Comment (shoc71) [reply dont stop now/assets/levels.txt:1380]: change them absolutely

**Comment by shoc71** [comment]
actually Imma close this and fix this in the new PR

**Pull request closed without merging**